src/clib/pio_getput_int.c

```diff
@@ -1016,45 +1025,44 @@ int PIOc_put_vars_tc(int ncid, int varid, const PIO_Offset *start, const PIO_Off
                      file->fh, varid));
                 pioassert(!start && !count && !stride, "expected NULLs", __FILE__, __LINE__);
 
-                /* Turn on independent access for pnetcdf file. */
-                if ((ierr = ncmpi_begin_indep_data(file->fh)))
-                    return pio_err(ios, file, ierr, __FILE__, __LINE__);
-
                 /* Only the IO master does the IO, so we are not really
                  * getting parallel IO here. */
                 if (ios->iomaster == MPI_ROOT)
                 {
                     switch(xtype)
                     {
                     case NC_BYTE:
-                        ierr = ncmpi_put_vars_schar(file->fh, varid, start, count, stride, buf);
```

Comment:
The calls here still happen from one proc, right? The buffered the calls need to be collective if you turn off indep data access, right?
(I also noted that the hints added in this pr expilicitly specify "no indep access")

Reply:
Wei-keng uses this style in his benchmark program. We can check with him for the answer.

Reply:
Thanks !
(The indep access stuff is left over from old code)